pkgs/tools/security/ibm-sw-tpm2/default.nix

```diff
@@ -9,6 +9,12 @@ stdenv.mkDerivation rec {
     hash = "sha256-PLZC+HGheyPVCwRuX5X0ScIodBX8HnrrS9u4kg28s48=";
   };
 
+  patches = [
+    # Backport openssl-3.1 gtom development branch.
```

Comment:
typo

Could you also list the upstream commit hash this has been cherry-picked from? Or fetchpatch from https://github.com/kgoldman/ibmswtpm2/commit/15501bf4973d334ca9420fa2fb0f0fe1800871e0 for example instead of vendoring the patch (possibly with includes= / excludes= if irrelevant parts conflict)

Reply:
Sounds good. Updated the PR.